File: src/main/java/com/ftn/mbrs/service/ModelVozilaService.java
package com.ftn.mbrs.service;

import java.util.List;

import com.ftn.mbrs.model.ModelVozila;

public interface ModelVozilaService {

	public ModelVozila save(ModelVozila modelVozila, Long tipPrikljuckaId, Long markaVozilaId);
	
	public ModelVozila update(ModelVozila modelVozila);

	public String delete(Long id);
	
	public ModelVozila findOne(Long id);
	
	public List<ModelVozila> findAll();
}

File: src/main/resources/static/js/services/ModelVozilaService.js
angular.module('gremoNaElektrikoApp.ModelVozilaService', [])
	.factory('ModelVozilaService', function($http) {
		var factory = {};
		
		factory.findAll = function() {
			return $http.get('/gremoNaElektriko/modelVozila');
		}
		
		factory.createModelVozila = function(modelVozila , tipPrikljuckaId, markaVozilaId) {
			return $http.post('/gremoNaElektriko/modelVozila'+'/'+tipPrikljuckaId+'/'+markaVozilaId, modelVozila);
		}

		factory.deleteModelVozila = function(id) {
			return $http.delete('/gremoNaElektriko/modelVozila/'+id);
		}
		
		factory.updateModelVozila = function(modelVozila) {
			return $http.put('/gremoNaElektriko/modelVozila', modelVozila);
		}
		
		return factory;
	});